scenarios/apim-baseline/terraform/README.md

```diff
@@ -0,0 +1,92 @@
+# Azure API Management - Secure Baseline [Terraform]
+
+This is the Terraform-based deployment guide for [Scenario 1: Azure API Management - Secure Baseline](../README.md).
+
+## Prerequisites
+
```

Comment:
I believe, one of the prerequisites is to have a terraform setup in the local machine.